### DISCUSSION TITLE
where is the old wiki?

### DISCUSSION THREAD

**Discussion by WilkAndy** [idaholab/moose#17614] [Q&A General] [answered]
I wanted to point @traiwit to some plasticity documentation on there, but can't find it

**Comment by aeslaughter** [comment]
It is dead, we don't have the resources to host it any longer, see  #15932

**Comment by hugary1995** [comment]
What specific information were you looking for? It might be somewhere in the new docs. If it is not, we should migrate them over.

**Comment by milljm** [comment, marked as answer]
@WilkAndy 
I turned it back on... So very temporary though:

  https://ec2-18-237-134-137.us-west-2.compute.amazonaws.com/

**Reply by WilkAndy** [reply]
Thank you @milljm and @aeslaughter .   This was very useful, and i just wanted to confirm that we got all the required doco.

**Reply by milljm** [reply]
👍 
I have shut down the AWS server...

**Comment by aeslaughter** [comment]
@milljm Re-enabled the old wiki. It will be up until the end of the day on Friday (4-16-2021). If you want something please grab the content and either submit a PR or paste the content into #15932. The content is very old, out-of-date, and it ends up costing personal money to host the site. After this, it is gone and whatever was "needed" should be replaced with new content.

**Reply by WilkAndy** [reply]
Thanks for this @aeslaughter and @milljm

**Reply by WilkAndy** [reply]
Oh, i see that last year i put my request into #15932 .   Did anyone ever action those requests (i didn't) ?

**Reply by WilkAndy** [reply]
I've downloaded everything i need, thank you @milljm .  It'd be appreciated if you could keep the AWS going for a few more hours to give @Traiwit a chance to download.

**Comment by WilkAndy** [comment]
@traiwit - i suggest you grab the following pages:

https://ec2-18-237-134-137.us-west-2.compute.amazonaws.com/wiki/PhysicsModules/TensorMechanics/HardSoft/
https://ec2-18-237-134-137.us-west-2.compute.amazonaws.com/wiki/PhysicsModules/TensorMechanics/ReturnMap/ (i think this is in the current doco too)
https://ec2-18-237-134-137.us-west-2.compute.amazonaws.com/wiki/PhysicsModules/TensorMechanics/CTO/
https://ec2-18-237-134-137.us-west-2.compute.amazonaws.com/wiki/PhysicsModules/TensorMechanics/WeakPlaneTensile/
https://ec2-18-237-134-137.us-west-2.compute.amazonaws.com/wiki/PhysicsModules/TensorMechanics/WeakPlaneShear/
https://ec2-18-237-134-137.us-west-2.compute.amazonaws.com/wiki/PhysicsModules/TensorMechanics/Tensile/
https://ec2-18-237-134-137.us-west-2.compute.amazonaws.com/wiki/PhysicsModules/TensorMechanics/MC/
https://ec2-18-237-134-137.us-west-2.compute.amazonaws.com/wiki/PhysicsModules/TensorMechanics/DP/
https://ec2-18-237-134-137.us-west-2.compute.amazonaws.com/wiki/PhysicsModules/TensorMechanics/TensorMechanicsBasics/PoroMechanics/

**Reply by Traiwit** [reply]
Hi @WilkAndy thank you much appreciated! I will go through them today :)
